.buildkite/lib/parseTemplate.ts

```diff
@@ -72,16 +72,16 @@ const validateFrontmatter = (meta: any): Frontmatter & { errors: string[] } => {
     errors.push("missing platforms");
   }
 
-  if (!meta.use_cases) {
-    errors.push("missing use_cases");
+  if (!meta.use_cases || meta.use_cases.length == 0) {
```

Comment:
I think we should be more strict with checking the use_cases. A possible human error may be adding a single use case as a string, and not an array of a single string.

Here's a defensive suggestion:

```suggestion
  if (!Array.isArray(meta.use_cases) || meta.use_cases.length == 0 || !meta.use_cases.every(use_case => typeof use_case === 'string')) {
```